web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -213,6 +215,13 @@ export default defineComponent({
       if (!valid && !sidebarOpen.value) {
         sidebarOpen.value = true;
       }
+
+      if (valid) {
+        snackbar.value = true;
+      } else {
+        snackbar.value = false;
+      }
```

Comment:
If I validate the current data harmonizer tab, I see the snackbar even if there is another tab with validation errors. This logic should look at all tabs and only trigger the snackbar if all tabs are valid.

Reply:
Ah, that's a good point. I have only been testing with 1 tab.

Reply:
@naglepuff I believe I've addressed this, I lifted the logic from `canSubmit` and applied to `snackbar`. I tested with some multi-tab data and it should behave properly.